iceoryx_posh/test/moduletests/test_roudi_service_registry.cpp

```diff
@@ -73,216 +168,195 @@ TEST_F(ServiceRegistry_test, AddMaximumNumberOfServiceDescriptionsWorks)
 
     for (auto& service : services)
     {
-        auto result = sut.add(service);
+        auto result = this->sut.add(service);
         ASSERT_FALSE(result.has_error());
     }
 }
 
-TEST_F(ServiceRegistry_test, AddMoreThanMaximumNumberOfServiceDescriptionsFails)
+TYPED_TEST(ServiceRegistry_test, AddMoreThanMaximumNumberOfServiceDescriptionsFails)
```

Comment:
I guess the correct test name is `AddServiceDescriptionsWhichWasAlreadyAddedAndReturnsOneResult`.